raiden/network/pathfinding.py

```diff
@@ -152,7 +152,8 @@ def get_pfs_info(url: str) -> PFSInfo:
             version=infos["version"],
         )
     except (json.JSONDecodeError, requests.exceptions.RequestException, KeyError) as e:
-        raise ServiceRequestFailed(str(e)) from e
+        msg = "Selected Pathfinding service returned unexpected reply"
```

Comment:
We seem to have three different notations of "pathfinding service". In the spec we call it "Pathfinding Service" in the Header in the text we note "path finding service" here it is "Pathfinding service". Can you make this consistent?

Reply:
Done